src/core/src/rt_info.cpp

```diff
@@ -74,7 +74,15 @@ ov::Any get_opset(const ov::Node::RTMap& rt_info) {
 
 void assign_runtime_info(const ov::Node::RTMap& from, ov::Node::RTMap& to) {
     auto opset = get_opset(to);
+
+    // Some items are designed to exist only in the original Node and disappear when copied to another one.
+    // This is done in case of items which are not applicable if their Node is replaced during the transformation
+    // pipeline.
+    std::vector<std::string> perishable_items = {"bin_offset", "original_size"};
```

Comment:
I think you need to define a class derived from RuntimeAttribute instead of manipulating with string names:
```
class OPENVINO_API ConstantOriginalSize: public ov::RuntimeAttribute {
public:
    OPENVINO_RTTI("ConstantOriginalSize");
    ConstantOriginalSize() = default;
    bool is_copyable() const override {
        return false;
    }
};
```

and I think having is_copiable == false will resolve the issue mentioned above.

Reply:
Yeah, that's definitely more elegant! Done.